TodoApp/TodoApp.Api/App_Start/WebApiConfig.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
+
+namespace TodoApp.Api
+{
+    public static class WebApiConfig
```

Comment:
I suggest renaming the file to `RouteConfig` and leaving the routes to be its only responsibility.